### ISSUE TITLE
Issues 1: Reader firmware upgrade seems to trigger automaticlly

### ISSUE THREAD

**Issue by shleeable** [sighmon/OpenDonation#8]
this might or might not break the flow... we need to detect if the reader is online and ready?

**Comment by sighmon** [comment]
Yeah I noticed that too - do you know if it's scheduled to do it at any time? Should we watch traffic too and from it for a bit to see whether it happens regularly?
